web/wp-content/themes/dctx-v2/assets/bower_components/jquery/src/traversing/var/rneedsContext.js

```diff
@@ -0,0 +1,8 @@
+define( [
+	"../../core",
+	"../../selector"
+], function( jQuery ) {
+	"use strict";
+
+	return jQuery.expr.match.needsContext;
+} );
```

Comment:
:no_entry_sign: **Error**: Closing parenthesis of a multi-line function call must be on a line by itself (*PEAR.Functions.FunctionCallSignature.CloseBracketLine*).